qermit/taskgraph/mittask.py

```diff
@@ -55,6 +55,22 @@ class IOTask(Enum):
     Dict,
 ]
 
+WireList = Union[
+    List[CircuitShots],
+    List[Circuit],
+    List[BackendResult],
+    List[ResultHandle],
+    List[AnsatzCircuit],
+    List[ObservableExperiment],
+    List[int],
+    List[float],
+    List[bool],
+    List[str],
+    List[QubitPauliOperator],
+    List[Dict[Qubit, Bit]],
+    List[Dict],
+]
+
```

Comment:
I don't know if there is a more succinct way of doing this. I would like `Tuple[List[Wire], ...]` to work but that assumes every `List[Wire]` in `Tuple` uses the same `Wire`.